etc/lmod-setup.sh

```diff
@@ -7,7 +7,7 @@ Usage: source etc/lmod-setup.sh PLATFORM
 
 OPTIONS:
    PLATFORM - name of machine you are building on
-      (e.g. cheyenne | hera | jet | orion | wcoss2 )
+      (e.g.  derecho | hera | jet | orion | wcoss2 )
```

Comment:
Was cheyenne left out from list on purpose?

Reply:
Set it back to "cheyenne". Derecho has another procedure of resetting modules, as added further down in lines 50-51.